workflow/rocoto/gfs_tasks.py

```diff
@@ -1461,13 +1466,20 @@ def gempak(self):
                      'maxtries': '&MAXTRIES;'
                      }
 
-        task = rocoto.create_task(task_dict)
+        fhrs = self._get_forecast_hours(self.cdump, self._configs['gempak'])
+        fhr_var_dict = {'fhr': ' '.join([f"{fhr:03d}" for fhr in fhrs])}
+
+        fhr_metatask_dict = {'task_name': f'{self.cdump}gempak',
+                             'task_dict': task_dict,
+                             'var_dict': fhr_var_dict}
+
+        task = rocoto.create_task(fhr_metatask_dict)
 
         return task
 
     def gempakmeta(self):
         deps = []
-        dep_dict = {'type': 'task', 'name': f'{self.cdump}gempak'}
+        dep_dict = {'type': 'metatask', 'name': f'{self.cdump}gempak'}
```

Comment:
@WalterKolczynski-NOAA, does this change the gempakmeta job to a metatask and run one forecast hour per task too?  The gempakmeta job runs less than 15 minutes, so I didn't make any changes.  It needs to process all forecast hours at once because gempak metafiles contain multiple lead times.

Reply:
No. This is the dependency for the `gempakmeta` job. `gempak` is a metatask now, so we had to change the type of all the jobs that depend on it.